.github/workflows/cd.branch-test.yml

```
@@ -169,10 +169,10 @@
 
       - name: Transfer files to server
         run: |
-          SERVER_DEST_BASE="${{ env.SERVER_USER }}@${{ env.SERVER_HOST }}:${{ env.DEPLOY_DIR }}"
+          SERVER_DEST_BASE="${{ env.SERVER_USER }}@${{ env.SERVER_HOST }}:${{ env.DEPLOY_DIR }}-test"
 
           # mkdir
-          ${{ env.SSH_CMD }} "mkdir -p ${{ env.DEPLOY_DIR }}" 
+          ${{ env.SSH_CMD }} "mkdir -p ${{ env.DEPLOY_DIR }}-test" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)